docs/UserGuide.md

```diff
@@ -389,25 +389,98 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 <div markdown="block" class="alert alert-danger">
 
-:exclamation: **Caution:** You cannot undo the sort once you have applied it!
+:exclamation: **Caution:** 
+* You cannot undo the sort once you have applied it! 
+* Sort will not affect the list permanently. Trackermon rearranges the show list one time after a successful sort command! 
+* After a successful sort command followed by an add command will still add the new show to the last index!  
```

Comment:
I think this could be rephrased to `A successful sort command followed by an add command will still result in the new show being added to the last index!`